dev/docker-compose.dev.yml

```diff
@@ -118,15 +121,15 @@ services:
             - redis
             - auth
             - cache
-            - postgresql
-    postgresql:
+            - postgres
+    postgres-test:
         image: postgres:11
         environment:
             - POSTGRES_USER=openslides
             - POSTGRES_PASSWORD=openslides
-            - POSTGRES_DB=openslides
+            - POSTGRES_DB=openslides-test
         networks:
-            - postgresql
+            - postgres
```

Comment:
maybe better to rename also to postgres-test!?

Reply:
Mmh, I don't like it. The network is still the same function-wise, only the container changed

Reply:
Change the names the way you prefer:-). This was my first attempt to fix this one, but then there have been more problems.